vizro-core/src/vizro/actions/_callback_mapping/_callback_mapping_utils.py

```diff
@@ -154,7 +154,7 @@ def _get_action_callback_inputs(action_id: ModelID) -> Dict[str, Any]:
             if "filter_interaction" in include_inputs
             else []
         ),
-        "theme_selector": (State("theme_selector", "on") if "theme_selector" in include_inputs else []),
+        "theme_selector": State("theme_selector", "on") if "theme_selector" in include_inputs else [],
```

Comment:
Debatable whether we really want to keep this key here at all or just not set it at all `if "theme_selector" not in include_input`, but I think current solution is ok.